targets/lenovo-x1/netvmExtraModules.nix

```diff
@@ -25,8 +25,11 @@
 
     internalNic = let
       vmNetworking = import ../../modules/microvm/virtualization/microvm/common/vm-networking.nix {
+        config = configH;
+        inherit lib;
         vmName = microvm.name;
         inherit (microvm) macAddress;
+        internalIP = 1;
       };
     in "${lib.head vmNetworking.networking.nat.internalInterfaces}";
 
```

Comment:
Element-vmIp static address different in the new hosts file.

Reply:
Should be fixed now.